go.mod

```diff
@@ -207,4 +207,4 @@ replace (
 	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.6.2
 )
 
-go 1.17
+go 1.19
```

Comment:
Bazel will still use 1.17. You have to update the WORKSPACE dependencies (rules_go and gazelle) and bump the golang version: https://github.com/bazelbuild/bazel-gazelle#running-gazelle-with-bazel.

Reply:
Ah, thank you, I knew it couldn't be that easy, working on it now!